ci/azure-cross-compile.yml

```diff
@@ -26,6 +34,14 @@ jobs:
         vmImage: ubuntu-16.04
         target: x86_64-sun-solaris
 
+      Linux_ARM:
```

Comment:
Is the name accurate when using android? I don't what, if any, differences there are still.

Reply:
i did not put much thought into the name...